themes/bootstrap5/templates/RecordDriver/ProQuestFSG/data-publicationDetails.phtml

```diff
@@ -0,0 +1,14 @@
+<?php foreach ($data as $field): ?>
+  <div>
+    <span<?=$this->schemaOrg()->getAttributes(['property' => 'publisher', 'typeof' => 'Organization'])?> class="format">
+      <?php $pubName = $field->getName(); ?>
+      <?php if (!empty($pubName)): ?>
+        <span<?=$this->schemaOrg()->getAttributes(['property' => 'name'])?>><?=$this->escapeHtml($pubName)?></span>
```

Comment:
@sturkel89 noted 
> Searches from clicking Publisher links in item records don't work

Unfortunately ProQuest's API does not support searching by publisher name, so I've removed this link altogether.

It *does* support searching by publication, i.e. journal title.  I've added that.